learning_service/Dockerfile

```diff
@@ -28,8 +28,7 @@ RUN pip install --upgrade pip wheel \
 # mllint tries to make its file executable - can't do this unless it's root or owns the file\
 RUN chown -R appuser: /usr/local/lib/python3.8/site-packages/mllint
 
-# TODO: Replace with actual application startup
-CMD ["bash"]
+CMD ["uvicorn", "src.main:app", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
The other application module already uses port 8000, if you run two of them in the Docker's host network mode, it will complain that the port is already taken. We could go with 8010 (cause the larger ones like 9000+ are usually reserved for the Prometheus metrics endpoints).